crates/sui-indexer/src/apis/governance_api.rs

```diff
@@ -39,7 +39,7 @@ impl GovernanceReadApiServer for GovernanceReadApi {
         self.fullnode.get_committee_info(epoch).await
     }
 
-    async fn get_sui_system_state(&self) -> RpcResult<SuiSystemState> {
+    async fn get_sui_system_state(&self) -> RpcResult<SuiSystemStateInnerV1> {
```

Comment:
any reason why this and other APIs need to return `SuiSystemStateInnerV1`?
I thought `SuiSystemState` was offering the right API and being able to be "destructed" into the Move object when you need to move it around